From ff0df9b346817c77ea053fb4349bce9bd95e7c28 Mon Sep 17 00:00:00 2001
From: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
Date: Wed, 20 Jun 2018 15:20:49 -0700
Subject: [PATCH 1226/1240] ASoC: uapi: sof: rename clk_id to mclk_id to make
 it explicit and fix the comment

Some platforms have more than one MCLK's exposed. So fix the comment
and rename the member to mclk_id to make it explicit to be used for
passing the corrent mclk id to the firmware.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
---
 include/uapi/sound/sof-ipc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/sound/sof-ipc.h b/include/uapi/sound/sof-ipc.h
index e3c3de0..f78b5cb 100644
--- a/include/uapi/sound/sof-ipc.h
+++ b/include/uapi/sound/sof-ipc.h
@@ -209,7 +209,7 @@ enum sof_ipc_dai_type {
 /* SSP Configuration Request - SOF_IPC_DAI_SSP_CONFIG */
 struct sof_ipc_dai_ssp_params {
 	uint16_t mode;   // FIXME: do we need this?
-	uint16_t clk_id; // FIXME: do we need this?
+	uint16_t mclk_id;
 
 	uint32_t mclk_rate;	/* mclk frequency in Hz */
 	uint32_t fsync_rate;	/* fsync frequency in Hz */
-- 
2.7.4

